frontend/src/pages/register/formValidation.tsx

```diff
@@ -25,7 +27,8 @@ const FormValidation = () => {
     handleSubmit,
     watch,
     formState: { errors },
-  } = useForm<FormData>({});
+  } = useForm<RegisterFormData>({});
```

Comment:
Why object literal in <RegisterFormData({})>?

Reply:
Removed, everything works without it.